### PR TITLE
Make "NFT Market" screen pixel perfect from Figma in React (Only UI)

### PR DESCRIPTION
**Pull request by bulatsalyakhovdev**
This PR deals with making "NFT Market" screen UI pixel perfect from Figma in React.

[Freedcamp 40026661](https://freedcamp.com/view/2956183/tasks/40026661)

image_2021_05_31

NOTE: It doesn't hook the page with backend to make it functional.

### CODE REVIEW

File: src/common/components/Header/HeaderScreen.tsx [hunk 1]
```diff
@@ -154,7 +154,7 @@ const Header = (props: PropsType) => {
             <Link to='#' className='flex items-center'>
               <img
                 src={addBtn}
-                className='w-20px h-20px mr-2px md:mr-8px'
+                className='w-20px h-20px mr-2px md:mr-2'
```
Comment (dbizhev) [inline src/common/components/Header/HeaderScreen.tsx:157]: @bulatsalyakhovdev can you change `mr-2` to `mr-2px`

Comment (bulatsalyakhovdev) [reply src/common/components/Header/HeaderScreen.tsx:157]: @dbizhev tailwind has several default spacing values, `mr-2` is `0.5rem`, which is `8px`. No need to add a `8px` in tailwind.config.js as it's duplicated

File: src/legacy/Routes.tsx [hunk 2]
```diff
@@ -109,11 +110,11 @@ class RouteApp extends React.Component<any, any> {
...
-    this.rpcRefreshIntervalId = window.setInterval(() => {
-      if (this.state.rpcConfig.username) {
-        rpc.refresh()
-      }
-    }, 10000)
+    // this.rpcRefreshIntervalId = window.setInterval(() => {
+    //   if (this.state.rpcConfig.username) {
+    //     rpc.refresh()
+    //   }
+    // }, 10000)
```
Comment (dbizhev) [inline src/legacy/Routes.tsx:117]: @bulatsalyakhovdev can you use `if (!app.isPackaged)`? Like this https://github.com/pastelnetwork/pastel-electron-wallet/blob/new_development/src/main.ts#L88

Comment (bulatsalyakhovdev) [reply src/legacy/Routes.tsx:117]: Hmm, this was not supposed to go on the commit. Is that the way to prevent it to happen on `new_development`?

**Review by br4in3x — CHANGES_REQUESTED** [review]
We need to set [minWidth for the BrowserWindow](https://github.com/electron/electron/blob/master/docs/api/browser-window.md#new-browserwindowoptions). This is how this screen looks if the window is shrunk:
image

Also, generally, the created layout is not matching the design. For comparison:
**Design**
image
**Created Layout**
image

**Comment by dbizhev** [comment]
> We need to set [minWidth for the BrowserWindow](https://github.com/electron/electron/blob/master/docs/api/browser-window.md#new-browserwindowoptions). This is how this screen looks if the window is shrunk:
> image
> 
> Also, generally, the created layout is not matching the design. For comparison:
> **Design**
> image
> **Created Layout**
> image

@br4in3x isn't minimum 900px? It's already set here https://github.com/pastelnetwork/pastel-electron-wallet/blob/new_development/src/main.ts#L73

And you should tell me what is not matching the design.

File: src/common/components/UpperSection/UpperSection.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (bulatsalyakhovdev) [reply src/common/components/UpperSection/UpperSection.tsx:1]: @br4in3x I renamed it in the PR [#113](https://github.com/pastelnetwork/pastel-electron-wallet/pull/113)

File: src/index.css [hunk 4]
```diff
@@ -27,28 +27,33 @@
...
-html {
-  overflow: hidden;
-}
-html,
-body,
-#root {
-  min-height: 100vh;
-  height: 100%;
+@layer components {
+  /* Used to wrapp main routes  */
```
[the remark this answers is not in the document]
Comment (bulatsalyakhovdev) [reply src/index.css:31]: @br4in3x what do you mean by "nit: type"?

**Comment by dbizhev** [comment]
@bulatsalyakhovdev I would suggest this PR should be reviewed after the PR #113 is merged into `new_development`

**Comment by bulatsalyakhovdev** [comment]
@br4in3x you should consider this PR just includes NFT pages, not whole app/project. The window's minimum width should be set to app/project globally, not any specific page.

**Review by dbizhev — APPROVED** [review]

File: src/common/components/UpperSection/UpperSection.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (bulatsalyakhovdev) [reply src/common/components/UpperSection/UpperSection.tsx]: @br4in3x I renamed it to "PageHeader".

**Comment by bulatsalyakhovdev** [comment]
> We need to set [minWidth for the BrowserWindow](https://github.com/electron/electron/blob/master/docs/api/browser-window.md#new-browserwindowoptions). This is how this screen looks if the window is shrunk:
> image
> 
> Also, generally, the created layout is not matching the design. For comparison:
> **Design**
> image
> **Created Layout**
> image

@br4in3x you should consider this PR just includes NFT pages, not whole app/project so that window's minimum width should be set to app/project globally, not any specific page/screen.

File: src/index.css [hunk 5]
```diff
@@ -27,18 +27,27 @@
...
 }
 
-@import '@fortawesome/fontawesome-free/css/all.css';
-@import 'typeface-roboto/index.css';
-@import '/src/common/assets/fonts/font.css';
-
-html {
-  overflow: hidden;
+@layer components {
+  /* It's used to wrapp main routes  */
```
Comment (ngvtuan) [inline src/index.css:31]: `wrapp`? is it `wrap` ?

**Review by ngvtuan — COMMENTED** [review]
Can we fix react warning? 
react-waring
[localhost-1622551345086.log](https://github.com/pastelnetwork/pastel-electron-wallet/files/6576328/localhost-1622551345086.log)


Please fix the UI issues
ui issues 1

ui issues 2

**Comment by bulatsalyakhovdev** [comment]
> Can we fix react warning?
> react-waring
> [localhost-1622551345086.log](https://github.com/pastelnetwork/pastel-electron-wallet/files/6576328/localhost-1622551345086.log)
> 
> Please fix the UI issues
> ui issues 1
> 
> ui issues 2

@ngvtuan fixed the UI issues and react warning as well.

**Comment by bulatsalyakhovdev** [comment]
> Can we fix react warning?
> react-waring
> [localhost-1622551345086.log](https://github.com/pastelnetwork/pastel-electron-wallet/files/6576328/localhost-1622551345086.log)
> 
> Please fix the UI issues
> ui issues 1
> 
> ui issues 2

@ngvtuan fixed UI issues and react warnings as well.

**Comment by bulatsalyakhovdev** [comment]
@ngvtuan @br4in3x we agreed to disregard the value "5/12" for now so that we can add it later once the designs/UX is updated.
image

**Comment by bulatsalyakhovdev** [comment]
@br4in3x I fixed the UI issues. Can you review again and approve the PR as well?

**Review by ArthurVogelsang — APPROVED** [review]

**Pull request merged**